about.html

```diff
@@ -0,0 +1,140 @@
+          Please contact us per Email for any further questions about World Snooker Championship 2022 <br />
+          <a href="#" class="text-center">worldsnooker2022@gmail.com</a>
+        </p>
+      </section>
+    </article>
+    <article class="about-logo d-flex flex-column justify-content-center align-items-center p-4 border-bottom">
+      <h2 class="section-heading p-3 text-center mb-0">
+        World Snooker Championship 2022 Logo
+      </h2>
+      <hr class="mt-0" />
+      <p class="text-center">
+        This logo was decided through the logo competition from 8.June to
+        7.July.
+      </p>
+      <section class="border m-2 mt-5 p-4 d-flex align-items-center shadow">
+        <div class="display-1 me-1 rounded-circle bg-warning p-3">
+          <img width="60px" height="80px" src="img/main-icon.png" alt="icon">
+        </div>
+        <div>
+          <h2 class="h2 font-weight-bold">
+            World Snooker <br />Championship 2022
+          </h2>
+          <p class="border-top border-bottom text-center text-muted">
+            16th April - 02nd May 2022 | Sheffield, England
+          </p>
+        </div>
+      </section>
+    </article>
+
+    <article class="event-gallery p-3 text-center d-flex flex-column">
+      <h2 class="section-heading">Watch the past tournaments</h2>
+      <hr class="align-self-center" />
+      <p class="p-3">
+        Take a look at the last two international snooker events which took
+        place in 2021 and 2018.
+      </p>
+      <section class="events p-5 d-flex flex-column flex-md-row gap-4" id="events-section">
+        <a href="#" class="event-card card">
+          <img height="100%" width="100%" src="img/snooker-event1.jpg" alt="chess event photo" class="card-image-top" />
+          <div class="card-img-overlay bg-danger bg-opacity-50 d-flex align-items-center justify-content-center">
+            <h3 class="display-3 text-center text-white">2021</h3>
+          </div>
+        </a>
+        <a href="#" class="event-card card">
+          <img src="img/snooker-event2.jpg" alt="chess event photo" class="card-image-top" />
+          <div class="card-img-overlay bg-danger bg-opacity-50 d-flex align-items-center justify-content-center">
+            <h3 class="display-3 text-center text-white">2018</h3>
+          </div>
+        </a>
+      </section>
+    </article>
+    <article class="partner-container d-flex flex-column align-items-center p-2" id="partner-section">
+      <h2 id="partner-heading" class="section-heading">Partners</h2>
+      <hr />
+      <section class="partner-grid d-flex flex-wrap align-items-center p-3 gap-4 justify-content-center">
+        <img src="img/partner1.png" alt="algorand logo" />
+        <img src="img/partner2.png" alt="algorand logo" />
+        <img src="img/partner3.png" alt="algorand logo" />
+        <img src="img/partner4.png" alt="algorand logo" />
+        <img src="img/partner5.png" alt="algorand logo" />
+      </section>
```

Comment:
- [ ] It would be nice to use a `ul` tag and have each partner wrapped in an `li` tag for semantic purposes, kindly do so